### PR TITLE
Cleaning content

### PR DESCRIPTION
**Pull request by polkhe**
formatted ExomeSeq to markdown. Remaining issue: | in formulas

### CODE REVIEW

File: Exome-Seq/tutorials/Diploid-variant-calling.md [hunk 1]
```diff
@@ -123,33 +142,42 @@ In this example we will perform variant calling and annotation using [genome in
...
+>
+> 3. Scrolling down to **Tweak algorithmic features?** click `Yes` and set **Calculate the marginal probability of genotypes and report as GQ in each sample field in the VCF output** to `Yes`. This would help us evaluating the quality of genotype calls:
+> 
+>   ![](../images/freebayes_gq.png)
+{: .hands_on}
 
 This produces a dataset in [VCF](http://www.1000genomes.org/wiki/Analysis/variant-call-format) format containing 35 putative variants. Before we can continue we need to post-process this dataset by breaking compound variants into multiple independent variants with **VcfAllelicPrimitives** tool found within **VCF Tools** section. This is necessary for ensuring the smooth sailing through downstream analyses:
 
-1. Select FreeBayes output as the input for this tool
+### :pencil2: Post-processing
```
Comment (bebatut) [inline Exome-Seq/tutorials/Diploid-variant-calling.md]: The hands-on box is missing :)

File: Exome-Seq/tutorials/Diploid-variant-calling.md [hunk 2]
```diff
@@ -189,78 +219,89 @@ family1    HG004_NA24143_mother -9                   -9                   2  1
...
 GEMINI database is queried using the versatile SQL language (more on SQL [here](http://swcarpentry.github.io/sql-novice-survey)). In Galaxy's version of GEMINI this is done using **GEMINI_query** tool. Within this tool SQL commands are typed directly into the **The query to be issued to the database** text box. Let's begin getting information from some of the tables we discovered with **GEMINI_db_info** tool above.
+
+> ### :bulb: Tip: Gemini tutorials
 >
-The examples below are taken from "[Intro to Gemini](https://s3.amazonaws.com/gemini-tutorials/Intro-To-Gemini.pdf)" tutorial. For extensive documentation see "[Querying GEMINI](http://gemini.readthedocs.org/en/latest/content/querying.html)".
+> The examples below are taken from "[Intro to Gemini](https://s3.amazonaws.com/gemini-tutorials/Intro-To-Gemini.pdf)" tutorial. For extensive documentation see "[Querying GEMINI](http://gemini.readthedocs.org/en/latest/content/querying.html)".
+{: .tip}
 
-#### :question: *Select "novel" variants that are not annotated in dbSNP database*
+### :pencil2: Selecting "novel" variants that are not annotated in dbSNP database
```
Comment (bebatut) [inline Exome-Seq/tutorials/Diploid-variant-calling.md]: Is it an hands-on box?

File: Exome-Seq/tutorials/Diploid-variant-calling.md [hunk 3]
```diff
@@ -189,78 +219,89 @@ family1    HG004_NA24143_mother -9                   -9                   2  1
...
- * `gt_types.subjectID` - three types of genotype types: `HOM_REF`, `HET`, `HOM_ALT`;
- * `gt_quals.subjectID` - genotype quality
- * `gt_depths.subjectID` - total number of reads in this subject at position
- * `gt_ref_depths.subjectID` -  number of reference allele reads in this subject at position
- * `gt_alt_depths.subjectID` - number of alternate allele reads in this subject at position
-
-#### :question: *At how many sites does child have a non-reference allele?*
-
-To answer this question we will use two fields of **GEMINI_query** interface:
+### :pencil2: Querying
```
Comment (bebatut) [inline Exome-Seq/tutorials/Diploid-variant-calling.md]: Is it an hands-on? The box must be missing

File: Exome-Seq/tutorials/Diploid-variant-calling.md [hunk 4]
```diff
@@ -273,28 +314,29 @@ This short tutorial should give you an overall idea on how generate variant data
...
-* Proceed with tutorial. For example, to repeat the following command from GEMINI tutorial:
-
-  ![](../images/gemini_command.png)
-
-* Use Galaxy's **GEMINI_load** tool:
-
-  ![](../images/galaxy_command.png)
-
-* and so on....
+> * Right click on the :notebook: symbol and open tutorial in a new browser tab;
```
Comment (bebatut) [inline Exome-Seq/tutorials/Diploid-variant-calling.md]: Is it inside a box here?

File: Exome-Seq/tutorials/Exome-Seq.md [hunk 5]
```diff
@@ -14,213 +19,255 @@ identify the genetic variants in all of an individual's genes. Exome sequencing
 is cheaper also than whole-genome sequencing. With a high coverage rate of
 100+ DP, 98% of all exons are covered.
 
-Items where exome sequencing can't identify genetic variation in:
+### Items where exome sequencing can't identify genetic variation in:
```
Comment (bebatut) [inline Exome-Seq/tutorials/Exome-Seq.md]: Why a title here?

File: Exome-Seq/tutorials/Exome-Seq.md [hunk 6]
```diff
@@ -14,213 +19,255 @@ identify the genetic variants in all of an individual's genes. Exome sequencing
...
+>
+> 1. [Data-preprocessing](#data-preprocessing)
+> 2. [Variant calling](#variant-calling)
+> 3. [Post-processing](#post-processing)
+> 4. [Variant analysis](#variant-analysis)
+{: .agenda}
+
+# Data-preprocessing
+
+> ### :nut_and_bolt: Comments
```
Comment (bebatut) [inline Exome-Seq/tutorials/Exome-Seq.md]: No need for a box here, I think

Comment (polkhe) [inline Exome-Seq/tutorials/Exome-Seq.md]: removed

File: Exome-Seq/tutorials/Exome-Seq.md (hunk 5 above)
Comment (polkhe) [inline Exome-Seq/tutorials/Exome-Seq.md]: removed :)

File: Exome-Seq/tutorials/Diploid-variant-calling.md (hunk 4 above)
Comment (polkhe) [inline Exome-Seq/tutorials/Diploid-variant-calling.md]: removed

File: Exome-Seq/tutorials/Diploid-variant-calling.md (hunk 3 above)
Comment (polkhe) [inline Exome-Seq/tutorials/Diploid-variant-calling.md]: added box

File: Exome-Seq/tutorials/Diploid-variant-calling.md (hunk 2 above)
Comment (polkhe) [inline Exome-Seq/tutorials/Diploid-variant-calling.md]: added box

File: Exome-Seq/tutorials/Diploid-variant-calling.md (hunk 1 above)
Comment (polkhe) [inline Exome-Seq/tutorials/Diploid-variant-calling.md]: Thanks! added box

**Comment by bgruening** [comment]
@polkhe great work! Thanks a lot!

**Pull request merged**